packages/transport-webrtc/src/muxer.ts

```diff
@@ -9,6 +9,7 @@ import type { Source, Sink } from 'it-stream-types'
 import type { Uint8ArrayList } from 'uint8arraylist'
 
 const PROTOCOL = '/webrtc'
+const DEFAULT_CHANNEL_CLOSE_DELAY = 30000
```

Comment:
How'd you arrive at this time? 30 seconds seems a bit long...

Reply:
I plucked it out of thin air.  It is configurable if users want something shorter.

Reply:
Gotcha, I suppose it will vary based on use case, and this is a temp fix anyway.